Cargo.toml

```
@@ -6,17 +6,19 @@ license = "MIT OR Apache-2.0"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
+[features]
+default = ["clap"]
+
 [dependencies]
 tokio = { version = "1.0", features = ["full"] }
 tracing = "0.1"
 tracing-subscriber = { version = "0.3", features = ["env-filter"] }
 reqwest = { version = "0.11.10", features = ["json", "native-tls-vendored"] }
 url = "2.2.2"
 http = "0.2.7"
-
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0.81"
 itertools = "0.10.3"
-
+clap = {version = "4.1.1", features = ["derive"], optional=true }
```

can we use the latest version?

also formatting

```suggestion
clap = {version = "4.3.10", features = ["derive"], optional = true }
```